content/documentation/pages/4-batch-developer-guides/4-continuous-deployment/1-cd-basics.md

```diff
@@ -0,0 +1,195 @@
+---
+path: 'batch-developer-guides/continuous-deployment/cd-basics/'
+title: 'Continuous Deployment of task applications'
+description: 'Continuous Deployment of Task applications'
```

Comment:
Update the description to be something different from the title.   i.e. This section will discuss how to use Continuous Deployment of Tasks in SCDF.
If you agree I can update on merge.

Reply:
Yeah, that is better. Please go ahead!